src/tests/ftest/nvme/nvme_pool_exclude.yaml

```diff
@@ -56,8 +56,8 @@ pool:
 container:
     type: POSIX
     control_method: daos
-    properties: cksum:crc16,cksum_size:16384,srv_cksum:on,rf:1
-    oclass: RP_2G8
+    properties: cksum:crc16,cksum_size:16384,srv_cksum:on,rf:2
+    oclass: RP_3G5
```

Comment:
Any idea why this is updated to new object class? (from RP_2G8 to RP_3G5).

Reply:
Yes, we need to guarantee the object is still accessible after excluding 2 replicas.